docs/2.7.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -180,6 +180,99 @@ Finally, you can ping the participant to see if the system is operational
 Version Specific Notes
 ~~~~~~~~~~~~~~~~~~~~~~
 
+Upgrade to Release 2.7
+^^^^^^^^^^^^^^^^^^^^^^
+Version 2.7 will slightly extend the database schema used. Therefore, you will have to perform the
+database migration steps.
+Alternatively, you can enable the new "migrate and start" mode of Canton, which will trigger an automatic update of the database schema
+when a new minor version is deployed.
+This mode can be enabled by setting the appropriate storage parameter:
+
+.. code:: bash
+
+    canton.X.Y.storage.parameters.migrate-and-start = yes
+
+In order to benefit from the new security features brought by protocol version 5,
+you will have to :ref:`upgrade the domain accordingly <canton_domain_protocol_version_upgrading>`.
+
+Activation of unsupported features
```

Comment:
@rgugliel-da is there a specific unsupported feature it is recommended to activate, or is this just a general note on the change in mechanics?

Reply:
The security features there: https://github.com/DACH-NY/canton/blob/6357b20b7125777ca0713c7248df59684a217c86/UNRELEASED.md#additional-security-validations

Reply:
I don't understand your comment Raf, the security enhancements are not opt-in, only a few are opt-out which caused problems with CN, but I wouldn't recomment that to clients to disable.

I think Brian's question still stands which unsupported features require the dev version support to be enabled.

Reply:
Not a big issue, but I wouldn't add "how to enable unsupported features" as part of the "how to migrate production systems" guide.

Reply:
Will drop.